backend/app/controllers/teams_profile_controller.rb

```diff
@@ -0,0 +1,22 @@
+class TeamsProfileController < ApplicationController
+  before_action :authenticate, only: [:index, :show]
+  before_action :set_user, only: [:index, :show]
+
+  def index
+    user = User.find(params[:user_id])
+    @teams = user.teams
```

Comment:
set_userメソッドでユーザーを取得しているので、
以下で良いかなと思います！

```suggestion
    @teams = @user.teams
```

jbuilerでsports_disciplinesとtarget_agesを@teamsから引っ張っており、N + 1になるので、preloadが必要かと思います！

Reply:
なるほど、分かりました。